extensions/amp-story/1.0/amp-story.css

```diff
@@ -924,6 +924,7 @@ amp-story .amp-video-eq,
   --i-amphtml-interactive-component-shadow: 0px 4px 12px rgba(60, 64, 67, 0.08), 0px 2px 4px rgba(60, 64, 67, 0.12);
   --i-amphtml-interactive-prompt-clamp: 4 !important;
   --i-amphtml-interactive-results-prompt-margin: 20px !important;
+  --i-amphtml-interactive-placeholder-background: rgba(32, 33, 36, 0.2) !important;
```

Comment:
nit: This might be more descriptive as:
`--i-amphtml-interactive-image-placeholder-background`

Reply:
As discussed in chat, we'll keep this as-is since it might be used for other contexts.